proto/batch_plan.proto

```diff
@@ -241,9 +241,10 @@ message PlanNode {
 message ExchangeInfo {
   enum DistributionMode {
     // No partitioning at all, used for root segment which aggregates query results
-    SINGLE = 0;
-    BROADCAST = 1;
-    HASH = 2;
+    DISTRIBUTION_UNSPECIFIED = 0;
```

Comment:
How about just naming it as `INVALID` without repeating the enum name to make them shorter?

Reply:
should zero value enum have the suffix UNSPECIFIED?

Reply:
Both LGTM. But we can strip the enum name as the prefix. 😄